src/Build/Logging/ParallelLogger/ParallelConsoleLogger.cs

```diff
@@ -535,6 +539,30 @@ public override void ProjectStartedHandler(object sender, ProjectStartedEventArg
                     WriteItems(e, e.Items);
                 }
             }
+
+            if (e.BuildEventContext == null || e.Items == null)
+            {
+                return;
+            }
+            // node and project context ids for the propertyOutputMap key
+            int nodeID = e.BuildEventContext.NodeId;
+            int projectContextId = e.BuildEventContext.ProjectContextId;
+
+            // Create the value to be added to the propertyOutputMap
+            StringBuilder logOutputProperties = new StringBuilder();
+
+            foreach (DictionaryEntry item in e.Items)
+            {
+                ITaskItem itemVal = (ITaskItem)item.Value;
+                // Determine if the LogOutputProperties item has been used
+                if (string.Equals((string)item.Key, "LogOutputProperties", StringComparison.OrdinalIgnoreCase))
+                {
+                    // Add the item value to the string to be printed in error/warning messages
+                    logOutputProperties.Append(" ").Append(itemVal.ItemSpec);
```

Comment:
It might be good to allow configuring the separator via a property (like `MSBuildProjectConfigurationDescriptionSeparator`). Default to space if none is set.

Reply:
That makes parsing harder so I think I'm against it.